v1/config/index.ts

```diff
@@ -0,0 +1,24 @@
+import { createAdmin, deleteAdmin, getAdmin, updateAdmin } from './admin'
+import { createPublisher, deletePublisher, getPublisher, updatePublisher } from './publisher'
+import { createSite, deleteSite, getSite, updateSite } from './sites'
+
+export default {
```

Comment:
Could these be reworked as classes?

E.g. `new Config(leveldb options here)` and the constructor doing stuff like `this.admin = new AdminConfig(leveldb instance)`

Reply:
yeah just wanted to stub this out for now but will refactor it when I start integrating leveldb

Reply:
Sounds good, we can refactor it when we introduce the API 👍